src/main/resources/view/MainWindow.fxml

```diff
@@ -12,7 +12,7 @@
 <?import javafx.scene.layout.VBox?>
 
 <fx:root type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1"
-         title="Address App" minWidth="450" minHeight="600" onCloseRequest="#handleExit">
+         title="TutorAid" minWidth="450" minHeight="600" onCloseRequest="#handleExit">
```

Comment:
Should we do this in a separate commit?

Reply:
I agree with this as well, might make more sense to make this change is a future commit since it is editing the MainWindow?

Reply:
Okay will revert this back. Thank you!